test/index.js

```diff
@@ -234,6 +234,124 @@ describe('Schwifty', () => {
         ]);
     });
 
+    describe('hpal command', () => {
+
+        it('correctly registers', async () => {
+
+            console._log = console.log;
+            const logOutput = [];
+
+            console.log = (...args) => {
+
+                logOutput.push(...args);
+            };
+
+            const migrationsDir = './test/migrations/generated';
+
+            // Remove migration file from any previous test run
+            Fs.readdirSync(migrationsDir)
+                .forEach((migrationFile) => {
+
+                    const filePath = Path.join(migrationsDir, migrationFile);
+
+                    if (filePath.includes('gitkeep')) {
+                        return;
+                    }
+
+                    Fs.unlinkSync(filePath);
+                });
+
+            const server = await getServer(getOptions({
+                migrationsDir,
+                models: [
+                    TestModels.Dog,
+                    TestModels.Person
+                ]
+            }));
+
+            expect(server.plugins.schwifty.commands && server.plugins.schwifty.commands.migrations).to.exist();
+
+            await server.initialize();
+            await server.plugins.schwifty.commands.migrations(server, []);
+            await server.stop();
+
+            expect(logOutput[0].includes('//////////////')).to.equal(true);
+            expect(logOutput[1].includes('Success!')).to.equal(true);
+            expect(logOutput[2].includes('Generated new migration file:')).to.equal(true);
```

Comment:
Would be good to move output from schwifty-migrations into the CLI command.  Will make schwifty-migrations much simpler to test, I'd guess!  schwifty-migrations can return the path to the generated file.